j5/components/piezo.py

```diff
@@ -16,18 +16,18 @@ class Note(Enum):
     notation and their related frequencies in Hz.
     """
 
-    C6 = 1046
-    D6 = 1174
-    E6 = 1318
-    F6 = 1396
-    G6 = 1567
+    C6 = 1047
```

Comment:
I initially wondered why these were changed, but I now see that the previous values were not correct.